cfspopcon/formulas/scrape_off_layer/two_point_model/upstream_SOL_collisionality.py

```diff
@@ -0,0 +1,31 @@
+"""Routine to calculate the upstream SOL collisionality."""
+
+from typing import Union
+
+import xarray as xr
+
+from ....unit_handling import Quantity, ureg, wraps_ufunc
+
+
+@wraps_ufunc(
+    return_units=dict(upstream_SOL_collisionality=ureg.dimensionless),
+    input_units=dict(separatrix_electron_density=ureg.m**-3, separatrix_electron_temp=ureg.eV, parallel_connection_length=ureg.m),
+)
+def calc_upstream_SOL_collisionality(
```

Comment:
Prefer `separatrix_SOL_collisionality` to be consistent with other variables like `separatrix_electron_temp`.